src/IIIFPresentation/API/Infrastructure/ControllerBaseX.cs

```diff
@@ -157,4 +159,37 @@ public static ObjectResult PresentationNotFound(this ControllerBase controller,
     {
         return controller.PresentationProblem(detail, null, (int)HttpStatusCode.NotFound, "Not Found");
     }
+
+    /// <summary>
+    /// Create an <see cref="ObjectResult"/> that produced a 403 response
+    /// </summary>
+    public static ObjectResult Forbidden(this ControllerBase controller)
+        => controller.PresentationProblem(statusCode: (int)HttpStatusCode.Forbidden);
+    
+    /// <summary>
+    /// Create a 201 result with standard serialized value or custom serialized IIIF object if value is
+    /// <see cref="JsonLdBase"/> 
+    /// </summary>
+    public static ActionResult PresentationCreated(this ControllerBase controller, string? uri, object? value)
+    {
+        if (value is JsonLdBase jsonLdBase)
+        {
+            try
+            {
+                controller.Response.Headers.Location = uri;
+                return new ContentResult
+                {
+                    Content = jsonLdBase.AsJson(),
+                    ContentType = IIIF.Presentation.ContentTypes.V3,
+                    StatusCode = 201,
+                };
+            }
+            catch (Exception ex)
+            {
+                var asdas = ex.Message;
```

Comment:
This doesn't seem to be used anywhere - should it be logging the message instead?

Reply:
My bad - was troubleshooting and missed this. Will remove.